src/align_func_proto.cpp

```diff
@@ -68,7 +110,31 @@ void align_func_proto(size_t span)
          {
             toadd = pc;
          }
-         as.Add(step_back_over_member(toadd));
+
+         if (pc->level > HOW_MANY_AS_LEVEL)
```

Comment:
I wonder how necessary this limitation is? Is this code sufficiently performance-sensitive that it justifies not just using a `std::vector` and ignoring the limit? (I guess that would need some additional refactoring though to dynamically create elements as they'd be used?)

Reply:
latter